carmajava/mock_drivers/src/main/java/gov/dot/fhwa/saxton/carma/mock_drivers/AbstractMockDriver.java

```diff
@@ -241,6 +244,9 @@ protected DriverStatus getDriverStatus() {
         case "lightbar":
           driverStatusMsg.setLightbar(true);
           break;
+        default:
+          log.warn("Function getDriverStatus received an unrecognized driver type: " + driverType);
+          break;
```

Comment:
Did you verify that mock drivers still run after this change.

Reply:
Yeah. The default case was not being used anyway I think.

Reply:
I think its used by the gps and obu drivers still, but if nothing is crashing it should be fine.